packages/components/Tabs/src/TabsItem.settings.macos.ts

```diff
@@ -0,0 +1,87 @@
+/*
+Disclaimer: these styles do not follow a specific figma design, but are
+meant to be placeholders until new designs are developed
+*/
+
+import { tabsItemName, TabsItemType } from './TabsItem.types';
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+import type { IViewProps } from '@fluentui-react-native/adapters';
+
+export const tabsItemSelectActionLabel = 'Select a TabsItem';
+
+export const settings: IComposeSettings<TabsItemType> = [
+  {
+    tokens: {
+      color: 'bodyText',
+      variant: 'subheaderStandard',
+      fontSize: 13,
```

Comment:
Do you have to specify font size? Is it not included with the variant?

Reply:
It is included in the variant, but there is no variant that has a fontSize of 13 which is what is specified in the Figma design.

Reply:
nit: I would probably push back on design and tell them that 13 isn't part of the set of pre-determined typography, and get a new size. If macOS has different font sizes than other platforms and we just haven't gotten around to updating things on our end, that's fine, but in general we should be avoiding using non-standard sizes

Reply:
For reference, I tried to follow the appe Human Interface Guidelines for typography when I made the apple theme and the typography variants https://developer.apple.com/design/human-interface-guidelines/macos/visual-design/typography/

Definitely not a blocker

Reply:
@ksiler @sriya112 sounds good to me! Thanks.